.obs/dockerfile/slem4r-base-iso/Dockerfile

```diff
@@ -0,0 +1,44 @@
+# SPDX-License-Identifier: Apache-2.0
+#!BuildTag: suse/sle-micro-rancher-iso/base-%%SLEMICRO_VERSION%%:latest
+#!BuildConstraint: hardware:disk:size unit=G 10
+#!BuildConstraint: hardware:memory:size unit=G 16
+
+ARG SLEMICRO_VERSION
+ARG SLE_VERSION
+
+FROM suse/sle-micro-rancher/base-${SLEMICRO_VERSION}:latest AS os
+FROM suse/sle-micro-rancher/${SLEMICRO_VERSION}:latest AS builder
+
+WORKDIR /iso
+
+COPY manifest.yaml manifest.yaml
+COPY --from=os / rootfs
+
+# Version value is taken form the elemental repository tags
+# Release value of this image and os image are unrelated
+RUN elemental --debug --config-dir . build-iso -o /output -n "sle-micro-rancher.$(uname -m)-base-%VERSION%-Build%RELEASE%" dir:rootfs
```

Comment:
In former iso images in OBS this %VERSION% is a placeholder to be replaced with the version computed from github sources, more specific form the `elemental.obsinfo` file. For that the `_service` file requires an obs_scm service and replace_with_package_version service. Something like:

```xml
  <service name="obs_scm">
    <param name="url">https://github.com/rancher/elemental</param>
    <param name="scm">git</param>
    <param name="include">.obs</param>
    <param name="extract">.obs/dockerfile/slem4r-base-iso/Dockerfile</param>
    <param name="revision">main</param>
    <param name="filename">elemental</param>
    <param name="versionformat">@PARENT_TAG@+git%cd.%h</param>
    <param name="versionrewrite-pattern">v([^-]+)-?.*(\+git.*)</param>
    <param name="versionrewrite-replacement">\1\2</param>
  </service>
  <service name="replace_using_package_version" mode="buildtime">
    <param name="file">Dockerfile</param>
    <param name="regex">%VERSION%</param>
    <param name="package">elemental</param>
    <param name="parse-version">patch</param>
  </service>
  ```

Reply:
Yes, this will be part of the package submission in OBS